moesifwsgi/config_manager.py

```diff
@@ -31,8 +46,11 @@ def check_and_update(self, response_etag):
         # Acquire a read lock and check if the configuration needs to be updated.
         # but ony if the last update was more than 5 minutes ago.
         with self._lock.gen_rlock():
-            if self.current_etag == response_etag or datetime.utcnow() < self.last_updated_time + timedelta(minutes=5):
-                return
+            if self.current_etag:
```

Comment:
nitpic: may be consolidate multiple if condition into a single one so that makes it's easier to read and maintain

Reply:
We can remove check for not null (first condition) otherwise, we could not consolidate it into one.